HomeHealthServices/R4/files/HomeHealthServicesRule-0.1.0.cql

```diff
@@ -0,0 +1,29 @@
+library HomeHealthServicesRule version '0.1.0'
+using FHIR version '4.0.0'
+include FHIRHelpers version '4.0.0' called FHIRHelpers
+
+parameter Patient Patient
+parameter service_request ServiceRequest
+
+define RULE_APPLIES:
+  true
+define PRIORAUTH_REQUIRED:
+  true 
```

Comment:
Does HHS need pre-auth? I thought only NEAT needs it.

Reply:
I'll remove it.